chrome_extension/background/httpauth.js

```diff
@@ -69,6 +69,14 @@ httpAuth.loginOrShowCredentials = function (logins) {
     }
     // no logins found
     else {
-        httpAuth.callback({});
```

Comment:
does that mean that if we don't know a given credential, we'll get these prompts on chrome, firefox AND safari?
this is not ok.